apps/retrospective/services/sprint_sync.go

```diff
@@ -165,8 +166,11 @@ func (service SprintService) SetSynced(sprintID uint) {
 }
 
 // SyncSprintMemberData ...
-func (service SprintService) SyncSprintMemberData(sprintMemberID string) (err error) {
+func (service SprintService) SyncSprintMemberData(sprintID uint, sprintMemberID string) (err error) {
```

Comment:
We should define the same order of params as it is being called with or vice-versa.

Reply:
Done